docs/pkg/client/events_query.md

```diff
@@ -0,0 +1,87 @@
+# Package `pocket/pkg/client/events_query`
+
+> An event query package for interfacing with cometbft and the Cosmos SDK, facilitating subscriptions to chain event messages.
+
+## Overview
+
+The `events_query` package provides a client interface to subscribe to chain event messages. It abstracts the underlying connection mechanisms and offers a clear and easy-to-use way to get events from the chain. Highlights:
+
+- Offers subscription to chain event messages matching a given query.
+- Uses the Gorilla WebSockets package for underlying connection operations.
+- Provides a modular structure with interfaces allowing for mock implementations and testing.
+- Offers considerations for potential improvements and replacements, such as integration with the cometbft RPC client.
+
+## Architecture Diagrams
+
+[Add diagrams here if needed. For the purpose of this mockup, we'll assume none are provided.]
+
+## Installation
+
+```bash
+go get github.com/yourusername/pocket/pkg/client/events_query
```

Comment:
```suggestion
go get github.com/pokt-network/poktroll/pkg/client/events_query
```
Any reason we are not doing this?

Reply:
This is only necessary for go modules external to `poktroll` who want to consume the `events_query` package as a library.